templates/registration/logout.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}
+{% block content %}
+<p>Logout</p>
+<a href="{% url 'login' %}">Click here to login</a>
```

Comment:
The link to the login page uses the URL name 'login'. Ensure that the URL pattern for login is correctly named 'login' in your URL configuration.